src/apps/search-result/search-result.tsx

```diff
@@ -76,9 +76,6 @@ const SearchResult: React.FC<SearchResultProps> = ({ q, pageSize }) => {
         {
           onSuccess: (campaign) => {
             setCampaignData(campaign);
-          },
-          onError: () => {
-            // TODO: when we handle errors - handle this error
```

Comment:
I talked to Mikkel about this yesterday, and it should be caught by our error boundary, but it is not considered a major error, so it would just be logged into the console.